images/base/files/etc/default/kubelet

```diff
@@ -1 +1 @@
-KUBELET_EXTRA_ARGS=--fail-swap-on=false
\ No newline at end of file
+KUBELET_EXTRA_ARGS=--runtime-cgroups=/system.slice/containerd.service
```

Comment:
This is the cgroup in which containerd operates (in kind, and in most cases since there's a standard in-tree service spec).

GKE nodes set this, noticed while debugging the runc issues, which reminded me that we aren't here.
So do the SIG node GCE containerd CI jobs.

https://github.com/kubernetes/test-infra/blob/6928f7825c26cd52dd2db97298db1882d6609584/config/jobs/kubernetes/sig-node/containerd.yaml#L20

https://github.com/kubernetes/kubernetes/blob/15a1f9a39db95a7f83ce5463a328566411d99d51/cluster/gce/util.sh#L823